block-production/tests/mod.rs

```diff
@@ -1,24 +1,28 @@
 use std::convert::TryInto;
 use std::sync::Arc;
 
-use parking_lot::RwLock;
-
 use beserial::Deserialize;
 use nimiq_block::{Block, BlockError, ForkProof};
 use nimiq_block_production::BlockProducer;
 use nimiq_blockchain::{AbstractBlockchain, Blockchain, PushError, PushResult};
-use nimiq_database::volatile::VolatileEnvironment;
+use nimiq_database::{
+    lmdb::{open as LmdbFlags, LmdbEnvironment},
+    volatile::VolatileEnvironment,
+};
 use nimiq_genesis::NetworkId;
 use nimiq_hash::{Blake2bHash, Hash};
 use nimiq_keys::{Address, KeyPair as SchnorrKeyPair, PrivateKey as SchnorrPrivateKey};
 use nimiq_primitives::coin::Coin;
 use nimiq_primitives::policy;
 use nimiq_test_utils::blockchain::{
-    fill_micro_blocks, sign_macro_block, sign_view_change, signing_key, voting_key,
+    fill_micro_blocks, fill_micro_blocks_with_txns, sign_macro_block, sign_view_change,
+    signing_key, voting_key,
 };
 use nimiq_transaction_builder::TransactionBuilder;
 use nimiq_utils::time::OffsetTime;
 use nimiq_vrf::VrfSeed;
+use parking_lot::RwLock;
+use tempdir::TempDir;
```

Comment:
We normally separate internal vs external dependencies... I would put them above as they were